packages/bpk-component-bottom-sheet/src/BpkBottomSheetInner-test.tsx

```diff
@@ -71,4 +71,20 @@ describe('BpkBottomSheetInner', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  it('should render correctly without title prop', () => {
+    const { asFragment } = render(
+      <BpkBottomSheetInner
+        id="my-bottom-sheet"
+        title=""
+        dialogRef={jest.fn()}
+        onClose={jest.fn()}
+        actionText="Dismiss"
+        onAction={jest.fn()}
+        exiting={false}
+      >
+        Bottom sheet content
+      </BpkBottomSheetInner>,
+    );
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Could you make this test a bit more perscriptive in checking if this is correct in the DOM?

Something like `expect(title).toBe("")` or similar